src/web/mjs/connectors/BilibiliManhua.mjs

```diff
@@ -9,6 +9,20 @@ export default class BilibiliManhua extends Connector {
         super.label = '哔哩哔哩 漫画 (Bilibili Manhua)';
         this.tags = [ 'manga', 'webtoon', 'chinese' ];
         this.url = 'https://manga.bilibili.com';
+
+        this.config = {
+            quality:  {
+                label: 'Preferred quality',
+                description: 'quality of images\nwebp is what the site displays as "raw"(lowest quality)\njpg is in the middle\npng is higher quality(and significantly larger file size)',
```

Comment:
Description might be misleading. The website will show optimized `webp` or `jpeg` (depending on browser support). I don't think such an extensive explanation is required, as most users probably know about these formats. Personally i would prefer the term `format` instead of `quality`.